python/pyspark/errors/error_classes.py

```diff
@@ -277,6 +277,11 @@
       "The UDTF '<name>' is invalid. It does not implement the required 'eval' method. Please implement the 'eval' method in '<name>' and try again."
     ]
   },
+  "INVALID_UDTF_RETURN_TYPE" : {
```

Comment:
I have the same error class defined here: https://github.com/apache/spark/pull/41989/files#diff-2823e146fc0e6bddff3505b5bee6e2b855782d9f71e900e6f9099fc97d1fffa6R280. I will change the error class name in my PR.